app/assets/stylesheets/darkswarm/_shop-filters.scss

```diff
@@ -173,22 +173,27 @@
   z-index: 90;
 }
 
-.filter-shopfront {
-  &.taxon-selectors, &.property-selectors {
-    background: transparent;
-  }
+.shop-filters-container {
+  position: sticky;
+  top: 65px;
 
-  // Shopfront taxons
-  &.taxon-selectors {
-    @include filter-selector($clr-blue, $clr-blue-light, $clr-blue-bright);
-  }
+  .filter-shopfront {
```

Comment:
What about `filter-shopfront` which are not nested to `shop-filters-container` ?

Reply:
oh yes I have not been careful enough, thanks !